src/random.cpp

```diff
@@ -255,6 +295,11 @@ void GetStrongRandBytes(unsigned char* out, int num)
     GetOSRand(buf);
     hasher.Write(buf, 32);
 
+    // Third source: HW RNG, if available.
+    if (GetHWRand(buf)) {
+        hasher.Write(buf, 32);
```

Comment:
the logging is now in `void RDRandInit()`, which is a better place, so I suppose this comment is outdated